scripts/type-generation/webdriverio-generate-typings.js

```diff
@@ -55,7 +59,7 @@ const gatherCommands = (commandPath, commandFile) => {
     const commandContents = fs.readFileSync(commandPath).toString()
     const commandDocs = dox.parseComments(commandContents)
     const commandTags = commandDocs[0].tags
-    const commandName = commandFile.substr(0, commandFile.indexOf('.js'))
+    const commandName = commandFile.substr(0, commandFile.indexOf('.js')).replace('$', '$$$')
```

Comment:
@christian-bromann ah, this was blowing my mind.

$$ command appeared in typing file as $. This is because of how replace works https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace

`$$ | Inserts a "$"`

Reply:
it was happening here: `templateContents.replace('// ... element commands ...', allElementCommands)`

so, `replace('$', '$$$')` replaces single $ with $$